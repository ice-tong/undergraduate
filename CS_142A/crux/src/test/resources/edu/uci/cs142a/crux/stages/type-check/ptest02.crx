// function can't have non int or bool arguments

func foo(a: int, b:void, c:error) : void {
}

func main() : void {
  ::println();
}
