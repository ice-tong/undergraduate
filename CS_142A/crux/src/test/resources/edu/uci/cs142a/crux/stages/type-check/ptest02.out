Error type-checking file.
TypeError(3,0)[Function foo has a void argument in position 1.]
TypeError(3,0)[Function foo has an error in argument in position 2: Unknown type: error.]
